tests/test_client_functional.py

```diff
@@ -1200,7 +1198,7 @@ def do_release():
     app.router.add_post('/', handler)
     client = await aiohttp_client(app)
 
-    with fname.open() as f:
```

Comment:
It'd be nice to move this to a separate PR.

Reply:
The test fails on azure but is passed on appveyor

Reply:
Okay, but I still think that it'd be nice to separate this and only keep the CI config here so that it'd be more atomic.

Reply:
I cannot test it in a separate PR, this is the problem.